src/events/routes/eventRoutes.ts

```diff
@@ -0,0 +1,8 @@
+import express from "express"
+import { getEvents } from "../controllers/getEvents"
+
+const router = express.Router()
+
+router.get("/getEvents", getEvents);
```

Comment:
why `getEvents` ? the naming of the route is not fully in line with RESTful convention
```suggestion
router.get("/events", getEvents);
```

- `/events` represents the resource (events) you are interacting with.
- `GET` defines the action (retrieving the list of events).